Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeViewController.swift

```diff
@@ -274,6 +274,9 @@ class CreateSafeViewController: UIViewController, UITableViewDelegate, UITableVi
             self.uiModel.setChain(chain)
             self.chain = self.uiModel.chain
             self.initExecutionBuilder()
+            if self.chain.isSupported(feature: .relayingMobile) {
```

Comment:
Not sure about this but maybe we need to introduce new state for the controller called "estimate relaying" same as "estimate" in which we get the relays left on parameter change

Reply:
Since we do not have unit tests for this class, I do not feel confident to rewrite this part right now.

Reply:
Agree, let's maybe create separated ticket for it please and proceed for now